_maps/shuttles/mining/mining_rad

```diff
@@ -0,0 +1,177 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"c" = (
+/obj/structure/shuttle/engine/propulsion/burst,
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"d" = (
+/obj/effect/spawner/structure/window/hollow/survival_pod/directional{
+	dir = 4
+	},
+/turf/open/floor/plating,
+/area/shuttle/mining)
+"h" = (
+/obj/effect/spawner/structure/window/hollow/survival_pod/directional{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/shuttle/mining)
+"l" = (
+/obj/machinery/light{
+	dir = 8
+	},
+/obj/structure/chair/fancy/shuttle{
+	dir = 4
+	},
+/turf/open/floor/plasteel/tech,
+/area/shuttle/mining)
+"o" = (
+/obj/structure/ore_box,
+/turf/open/floor/plasteel/tech/grid,
+/area/shuttle/mining)
+"p" = (
+/obj/machinery/door/airlock/survival_pod/glass,
+/obj/docking_port/mobile{
+	dir = 8;
+	dwidth = 3;
+	height = 5;
+	id = "mining";
```

Comment:
these vars need to be absolutely perfect.

dwidth is the horizontal offset OF the docking port from the bottom left corner from the perspective of the docking port's facing dir, height is the vertical offset that the ship ends from the perspective of the docking port, width is the same but horizontal. port_direction and dir determine how it lands

make sure this works

Reply:
See: https://wiki.beestation13.com/view/Guide_to_mapping#Dwidth_and_Dheight_in_more_depth